bitnami/etcd/templates/scripts-configmap.yaml

```diff
@@ -245,8 +246,9 @@ data:
     exec 3>&1
     exec 4>&2
 
-    if [ "${BITNAMI_DEBUG}" = true ]; then
+    if [ "${BITNAMI_SNAPSHOT_DEBUG}" = true ]; then
```

Comment:
Hi, I would suggest you to leave this environment variable as `BITNAMI_DEBUG`, it is our standard env var to enable debug.

Reply:
I am receiving conflicting request from @rafariossaa and @miguelaeh [here](https://github.com/bitnami/charts/pull/1958#pullrequestreview-363271624). I am partial towards going with @miguelaeh, but if you have a discussion and come to a consensus, please let me know.

Reply:
PR #1958 that implemented this and enabled debug for either `BITNAMI_DEBUG` or `BITNAMI_SNAPSHOT_DEBUG` was merged this morning, so this is now solved.